tmt/steps/provision/testcloud.py

```diff
@@ -352,13 +317,6 @@ def _guess_image_url(self, name: str) -> str:
             raise ProvisionError(f"Could not map '{name}' to compose.")
         return url
 
-    @staticmethod
-    def _create_template() -> None:
-        """ Create libvirt domain template """
-        # Write always to ovewrite possible outdated version
-        with open(DOMAIN_TEMPLATE_FILE, 'w') as template:
-            template.write(DOMAIN_TEMPLATE)
-
     def wake(self) -> None:
```

Comment:
FYI, this method isn't ported yet. I'd probably do that in a following PR as it'd require splitting start to be able to execute code responsible for constructing DomainConfiguration to be able to call it there too.

If you prefer otherwise, I can handle it as a part of this PR. Also, I hadn't figured out how to actually trigger this codepath, am I blind or is it just a placeholder fn?

Reply:
The `wake()` method is used to connect to a running instance when using an existing tmt run, for example like this:

```
tmt run provision -h virtual
tmt run --last login
tmt run --last finish
```

It seems that everything needed to connect to the instance is working nicely so I see no blocker here.

Reply:
Thanks for the way to call the code, will address in a future PR targeting 1.28.

Reply:
Great, thanks!